platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -283,9 +283,9 @@ index 4babe476f..8d41a7690 100644
  import android.telephony.TelephonyManager;
  import android.util.Log;
  
-+// -- GODOT start --
++// -- BLAZIUM start --
 +import android.annotation.SuppressLint;
```

Comment:
This line maybe from a different branch, please validate this is required for rebranding